### PR TITLE
fix community fee displayed value

### LINKED ISSUE #1414: Convert Community Fee from decimal to percentage
We need to convert the community Fee from decimal to percentage. It should show 0.1% instead of 0.001%

image

### PR DESCRIPTION
**Pull request by claudiaHash**
Fixes #1414  .

### CODE REVIEW

File: src/components/Publish/Pricing/Fees.tsx [hunk 1]
```diff
@@ -43,13 +44,17 @@ export default function Fees({
   pricingType: 'dynamic' | 'fixed'
 }): ReactElement {
   const [field, meta] = useField('pricing.swapFee')
-  const [opcFees, setOpcFees] = useState<OpcFeesData>(undefined)
+  const [communityFee, setCommunityFee] = useState<string>('')
```
Comment (kremalicious) [inline src/components/Publish/Pricing/Fees.tsx:47]: good thinking making variable consistent to what we display as this stays confusing in code (like, `response.swapOceanFee` is the "community fee" coming from `opcFees` query). But this raises the question what exact "community fee" this actually is and f we should update the UI label too.

Based on [our docs](https://docs.oceanprotocol.com/concepts/fees/#ocean-community-fee), there is a "Ocean Community swap fee" and a "Ocean Community order fee". If we use those 2 distinctions in the docs then we need to have this distinction in our UI too

**Comment by EnzoVezzaro** [comment]
found a little thing. Maybe it's even worth doing anything as it doesn't impact anything, it's just a weird little thing 🙃 

if you're not connected, the community fee is '0'

Screen Shot 2022-05-11 at 9 42 57 AM

**Comment by EnzoVezzaro** [comment]
found a little thing. Maybe it's not even worth doing anything as it doesn't impact anything at all, it's just a weird thing to see 🙃

if you are not connected, the amount shown is '0'

Screen Shot 2022-05-11 at 9 42 57 AM

**Comment by claudiaHash** [comment]
> if you are not connected, the amount shown is '0'

hmmm, don't think it's introduced by the changes done here since I can reproduce it on main v4, but looking into fixing this

File: src/components/Publish/Pricing/Fees.tsx (hunk 1 above)
Comment (mihaisc) [reply src/components/Publish/Pricing/Fees.tsx:47]: yep , we need both swap and consume (this is how they are called in the sub query)

Comment (mihaisc) [reply src/components/Publish/Pricing/Fees.tsx:47]: aaaand i see a redesign incoming 😆

**Comment by kremalicious** [comment]
> if you are not connected, the amount shown is '0'

first guess is that we get those fees with web3 call now, which is fine but we do not fall back to our "dummy Web3", meaning Infura. Noticed this in the ENS PR and best solution would be to adapt our Web3 context to do that automatically. Otherwise we end up having to handle the web3 fallback in multiple places. So would say this should not be done in this PR to keep it light

**Comment by kremalicious** [comment]
And if we want to get this PR with original number-only fix through, we should at least update the label we show too, then handle adding the "Ocean Community order fee" in another PR, as this most likely requires layout or design changes

**Review by kremalicious — APPROVED** [review]

**Pull request merged**